traffic_ops/testing/api/v4/parameters_test.go

```diff
@@ -141,7 +179,78 @@ func UpdateTestParameters(t *testing.T) {
 	if respParameter.Value != expectedParameterValue {
 		t.Errorf("results do not match actual: %s, expected: %s", respParameter.Value, expectedParameterValue)
 	}
+	//update back to old value for safe deletion in pre-requisite
+	alert, _, err = TOSession.UpdateParameter(remoteParameter.ID, old, nil)
+	if err != nil {
+		t.Errorf("cannot UPDATE Parameter by id: %v - %v", err, alert)
+	}
+}
+
+func UpdateParametersInvalidValue(t *testing.T) {
```

Comment:
Is this updating a Parameter with an "Invalid Value"? It seems like the expectation is for the update to succeed

Reply:
changed the function name to UpdateParametersEmptyValue - updating parameters can take empty value. so the tests are succeed.